RomanPort.PCParadiseBot/Program.cs

```diff
@@ -14,7 +16,7 @@ class Program
     {
         public static DiscordClient discord;
         public static List<PCModule> modules;
-        
+        public static RedditClient reddit;
```

Comment:
You should store this as a static variable inside of your own module, not in Program.cs

Reply:
Fixed.